benchmarks/dbscan/data.hpp

```diff
@@ -27,7 +27,9 @@ std::vector<Point<DIM>> sampleData(std::vector<Point<DIM>> const &data,
   std::vector<Point<DIM>> sampled_data(num_samples);
 
   std::default_random_engine generator(17);
-  std::uniform_int_distribution distribution;
+  // FIXME_NVCC(11.0-11.3): not providing int here results in an internal
+  // compiler error
+  std::uniform_int_distribution<int> distribution;
```

Comment:
I tried to protect this with cuda compiler check and to
```c++
defined(KOKKOS_COMPILER_NVCC) && (KOKKOS_COMPILER_NVCC < 1140)
  using uniform_int_distribution = std::uniform_int_distribution<int>;
#else
  using uniform_int_distribution = std::uniform_int_distribution;
#endif
```
But that still crashed the compiler.